examples/art-boards/composable-apps/chat-vue/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@art-boards/chat-app-vue",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "lean dev --config=vue",
+    "build": "lean build --config=vue",
+    "test": "echo 'ToDo'",
+    "lint": "tsc --noEmit",
+    "deploy:pr": "lean deploy"
+  },
+  "main": "./src/index.ts",
+  "author": "",
+  "license": "MIT",
+  "peerDependencies": {
+    "@leanjs/core": "*",
+    "@leanjs/vue": "*",
+    "vue": "*"
```

Comment:
We also need `vue` as a `dependency` in the root package.json

Reply:
Yep - done that now :+1: